src/table/table.component.ts

```diff
@@ -359,6 +361,14 @@ export class Table implements AfterViewInit, OnDestroy {
 		}
 	}
 
+	/**
+	 * Setting sortable to false will disable all headers including headers which are sortable. Is is
+	 * possible to set the sortable state on the header item to disable/enable sorting for only some headers.
+	 */
+	@Input() sortable = true;
+
+	@Input() noBorder = true;
```

Comment:
support for a table rendered without a border? Try to make sure to document new provisions